.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push, workflow_dispatch]
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-python@v4
+
+    - name: install oj-verify
+      run: pip3 install -U online-judge-verify-helper
+
+    - name: Set up Rust (nightly)
+      run: rustup install nightly
+
+    - name: cargo fmt
+      run: cargo fmt --all -- --check
+
+    - name: use oj-verify
+      with:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
needed so verify helper tool can push test results to that `timestamp.remote` file